<ui version="4.0" >
 <class>TabBookmarks</class>
 <widget class="QWidget" name="TabBookmarks" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QListWidget" name="list" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAdd" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit" >
       <property name="text" >
        <string>Edi&amp;t</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDel" >
       <property name="text" >
        <string>&amp;Del</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
